src/components/ScholarshipForm.jsx

```diff
@@ -45,8 +45,8 @@ function ScholarshipForm({ scholarship }) {
   const navigate = useNavigate();
 
   const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'));
-  const { t } = useTranslation(['scholarshipForm', 'common', 'validation']);
-  const { t: validationT } = useTranslation('validation');
+  const { t: validationT } = useTranslation('validations');
```

Comment:
```suggestion
const { t: validationT } = useTranslation('validation');
```
We can also change the namesapce name to validations.

Reply:
Whoops, nice catch! I'm surprised our validation checker didn't catch it 🤔